src/main/java/org/reaktivity/nukleus/http2/internal/stream/Http2ServerFactory.java

```diff
@@ -1599,8 +1648,12 @@ private void onDecodeError(
             long authorization,
             Http2ErrorCode error)
         {
-            streams.values().forEach(ex -> ex.cleanup(traceId, authorization));
-            doEncodeGoaway(traceId, authorization, error);
+            this.traceId = traceId;
+            this.authorization = authorization;
+            this.error = error;
```

Comment:
Rename `this.error` to `this.decodeError`.
Ideally we would not need this instance state and could send with the signal as a `Http2SignalEx`.